Source code examples give linter issues

The source code examples give various linter warnings using the golangci-lint tool:

```
$ golangci-lint run --no-config --deadline=1000s -E=golint -E=gocyclo -E=maligned -E=prealloc -E=unconvert -E=gosec -E=whitespace -E=misspell
system-configuration\files\directoryListSafe.go:30:21: Error return value of `http.ListenAndServe` is not checked (errcheck)
        http.ListenAndServe(":8080", http.StripPrefix("/tmp/static", http.FileServer(fs)))
                           ^
access-control\URL.go:29: unnecessary trailing newline (whitespace)

}
access-control\URL.go:62: unnecessary leading newline (whitespace)
        return http.HandlerFunc(func(res http.ResponseWriter, req *http.Request) {

access-control\URL.go:117: unnecessary trailing newline (whitespace)

}
file-management\filetype\filetype.go:43: unnecessary trailing newline (whitespace)

}
access-control\URL.go:124:2: S1023: redundant `return` statement (gosimple)
        return
        ^
```

--- a/src/access-control/URL.go
+++ b/src/access-control/URL.go
@@ -26,7 +26,6 @@ func Homepage(res http.ResponseWriter, req *http.Request) {
 	fmt.Fprintf(res, "<a href=\"login\"> Login </a> <br>")
 	fmt.Fprintf(res, "<a href=\"profile?page=page2\"> URL Parameters page 2 - Protected </a> <br>")
 	fmt.Fprintf(res, "<a href=\"profile\"> Profile - Protected </a> <br>")
-
 }
 
 // create a JWT and put in the clients cookie
@@ -59,8 +58,7 @@ func setToken(res http.ResponseWriter, req *http.Request) {
 
 // middleware to protect private pages
 func validate(page http.HandlerFunc) http.HandlerFunc {
-	return http.HandlerFunc(func(res http.ResponseWriter, req *http.Request) {
-
+	return func(res http.ResponseWriter, req *http.Request) {
 		cookie, err := req.Cookie("Auth")
 		if err != nil {
 			res.Header().Set("Content-Type", "text/html")
@@ -91,7 +89,7 @@ func validate(page http.HandlerFunc) http.HandlerFunc {
 			fmt.Fprintf(res, "<a href=\"login\"> Login </a>")
 			return
 		}
-	})
+	}
 }
 
 // only viewable if the client has a valid token
@@ -114,14 +112,12 @@ func protectedProfile(res http.ResponseWriter, req *http.Request) {
 		fmt.Fprintf(res, "Hello %s <br>", claims.Username)
 		fmt.Fprintf(res, "<a href=\"logout\"> Logout </a>")
 	}
-
 }
 
 // deletes the cookie
 func logout(res http.ResponseWriter, req *http.Request) {
 	deleteCookie := http.Cookie{Name: "Auth", Value: "none", Expires: time.Now()}
 	http.SetCookie(res, &deleteCookie)
-	return
 }
 
 func main() {

--- a/src/file-management/filetype/filetype.go
+++ b/src/file-management/filetype/filetype.go
@@ -40,5 +40,4 @@ func main() {
 	default:
 		fmt.Println("unknown file type uploaded")
 	}
-
 }

--- a/src/system-configuration/files/directoryListSafe.go
+++ b/src/system-configuration/files/directoryListSafe.go
@@ -1,6 +1,7 @@
 package main
 
 import (
+	"log"
 	"net/http"
 	"os"
 )
@@ -27,5 +28,8 @@ func (f neuteredReaddirFile) Readdir(count int) ([]os.FileInfo, error) {
 
 func main() {
 	fs := justFilesFilesystem{http.Dir("tmp/static/")}
-	http.ListenAndServe(":8080", http.StripPrefix("/tmp/static", http.FileServer(fs)))
+	err := http.ListenAndServe(":8080", http.StripPrefix("/tmp/static", http.FileServer(fs)))
+	if err != nil {
+		log.Fatal("ListenAndServe: ", err)
+	}
 }

